app/common/renderer/actions/Inspector.js

```diff
@@ -368,6 +369,20 @@ export function setSessionDetails({driver, sessionDetails, mode, mjpegScreenshot
   };
 }
 
+export function storeSessionSettings(sessionSettings = null) {
+  return async (dispatch, getState) => {
+    if (sessionSettings === null) {
+      const action = applyClientMethod({
+        methodName: 'getSettings',
+        skipRefresh: true,
+        ignoreResult: true,
+      });
+      sessionSettings = await action(dispatch, getState);
```

Comment:
try to avoid reassigning function arguments. It is a suboptimal coding practice.